kit/src/main/java/com/oracle/javafx/scenebuilder/kit/metadata/Metadata.java

```diff
@@ -5412,6 +5419,7 @@ private Metadata() {
         MenuBarMetadata.getProperties().add(accessibleRole_MENU_BAR_PropertyMetadata);
         MenuBarMetadata.getProperties().add(menusPropertyMetadata);
         MenuBarMetadata.getProperties().add(styleClass_c18_PropertyMetadata);
+        MenuBarMetadata.getProperties().add(useSystemMenuBarNamePropertyMetadata);
```

Comment:
`systemMenuBar` is added to `hiddenProperties` below. Probably you need to remove that.

Reply:
It's been removed. Interestingly it worked even without removing it.